addons/libkosext2fs/fs_ext2.c

```diff
@@ -1583,7 +1583,10 @@ static int fs_ext2_symlink(vfs_handler_t *vfs, const char *path1,
     /* Will the link fit in the inode? */
     if(len < 60) {
         /* We can make a fast symlink. */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wstringop-truncation"
         strncpy((char *)inode->i_block, path1, 60);
+#pragma GCC diagnostic pop
```

Comment:
So your "fix" is to hide the bug?

Reply:
Yes and no. There is another "fix" in this PR that does the same for VMU filename where there is no NULL terminator. I made the same "fix" here just to bring focus to the part of the code that generates the warning because I know BlueCrab ,  who wrote this code, would know if should suppress it or actually "fix" it.